### PR TITLE
Additional logging of url, path and data

### LINKED ISSUE #4822: Control the source of content data between Live/Test for TEST MAP pages
**Is your feature request related to a problem? Please describe.**
We want to be able to control whether we use PROD or TEST content data for MAP pages on the TEST environment. This is going to require additional mechanism being added to Mozart.

**Describe the solution you'd like**
Simorgh will accept a URL param to specify the source environment of content data (e.g. TEST or LIVE) and will pass this to Mozart. This will allow us to load TEST content or LIVE content based on this control parameter.

`renderer_env=int|test|live` e.g. http://www.example.com/pidgin/12345?renderer_env=live

NOTE Live Simorgh should never be able to load TEST content.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by karinathomasbbc**
Resolves #4822

**Overall change:** 
Log the url, path and data endpoint for diagnostic purposes 

**Code changes:**
- Log the data endpoint
- Log the path (excludes query string) & url (includes query string)

Result:
```
info [server/index.jsx] Path: [/pidgin/23248703] URL: [/pidgin/23248703?a=b]
info [getInitialData/index.js] Data: [http://localhost:7080/pidgin/23248703.json]
```
---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/routes/getInitialData/index.js [hunk 1]
```diff
@@ -18,7 +18,11 @@ const baseUrl = onClient()
   ? getBaseUrl(window.location.origin)
   : process.env.SIMORGH_BASE_URL;
 
-const getUrl = pathname => `${baseUrl}${pathname.replace(ampRegex, '')}.json`;
+const getUrl = pathname => {
+  const url = `${baseUrl}${pathname.replace(ampRegex, '')}.json`;
+  logger.info(`Data: [${url}]`);
```
Comment (chris-hinds) [inline src/app/routes/getInitialData/index.js]:
```suggestion
  logger.info(`DataRequest: [${url}]`);
```

Comment (jamesdonoh) [inline src/app/routes/getInitialData/index.js]: Would it make any difference if we put this after L67? I wouldn't usually expect a function called `getXXX` to  have side-effects.

Comment (karinathomasbbc) [reply src/app/routes/getInitialData/index.js]: 👍

Comment (karinathomasbbc) [reply src/app/routes/getInitialData/index.js]: Good shout, I have moved it out of getUrl and into the fetchData function instead

**Review by chris-hinds — APPROVED** [review]

**Review by ryanmccombe — APPROVED** [review]

**Pull request merged**